src/csexp_rpc/io_buffer.mli

```diff
@@ -31,3 +31,15 @@ val pos : t -> int
 
 (** [length t] the number of bytes to read [bytes t] *)
 val length : t -> int
+
+val write_pos : t -> int
+
+val commit_write : t -> len:int -> unit
+
+val max_write_len : t -> int
+
+val read_char_exn : t -> char
+
+val read_into_buffer : t -> Buffer.t -> max_len:int -> int
+
+val maybe_resize_to_fit : t -> int -> unit
```

Comment:
Just adding small comments explaining these functions is what I had in mind. So it is more uniform with the rest of the file. For example what is "commit_write"?